source/_integrations/velbus.markdown

```diff
@@ -99,6 +99,24 @@ Use this service when you make changes to your configuration via velbuslink.
 | `interface`            | no       | The port used to connect to the bus (the same one used during configuration). |
 | `address`              | no       | The module address in decimal format, which is displayed on the device list on the integration page, if provided the service will only clear the cache for this model, without an address, the full velbuscache will be cleared. |
 
+
+## VMB7IN and the Energy dashboard
+
+The VMB7IN does not repport what the counter is counting, because of this we need to let Home Assistant know what device class each counter is, in case the counter is counting an energy device everything will work out of the box.
+But in case the VMB7IN channel is a water or gas counter you will need to make some modifications in your configuration.yaml.
+
+```yaml
+homeassistant:
+  customize:
+    sensor.eau_counter:
```

Comment:
is this called `eau_counter` both for water and gas?

Reply:
no this name can be whatever they configurd during the setup of there velbus system.